### ISSUE TITLE
bcftools 1.10.2 fails to build, missing dependency libhts-dev==1.10.2

### ISSUE THREAD

**Issue by kjsanger** [wtsi-npg/npg_conda#182]
```
ERROR:root:    raise ResolvePackageNotFound(bad_deps)
ERROR:root:conda.exceptions.ResolvePackageNotFound:
ERROR:root:  - libhts-dev==1.10.2
```

**Comment by kjsanger** [comment]
Caused by corrupt channel index?

**Comment by kjsanger** [comment]
Resolved by #191
